src/main/java/com/gregtechceu/gtceu/api/fluids/store/FluidStorageKeys.java

```diff
@@ -13,13 +13,13 @@ public final class FluidStorageKeys {
 
     public static final FluidStorageKey LIQUID = new FluidStorageKey(GTCEu.id("liquid"),
             MaterialIconType.liquid,
-            m -> prefixedRegistredName("liquid.", FluidStorageKeys.LIQUID, m),
+            m -> prefixedRegistredName("liquid_", FluidStorageKeys.LIQUID, m),
             m -> m.hasProperty(PropertyKey.DUST) ? "gtceu.fluid.liquid_generic" : "gtceu.fluid.generic",
             FluidState.LIQUID, 0);
 
     public static final FluidStorageKey GAS = new FluidStorageKey(GTCEu.id("gas"),
             MaterialIconType.gas,
-            m -> prefixedRegistredName("gas.", FluidStorageKeys.GAS, m),
+            m -> prefixedRegistredName("gaseous_", FluidStorageKeys.GAS, m),
```

Comment:
`prefixedRegistredName` has a typo
also I think this should actually be a postfix of `"_gas"` instead.